ethical_statement.md

```diff
@@ -0,0 +1,35 @@
+## Preamble
+
+OpenDataCam (ODC) was created as an open-source tool to quantify the world. It is designed to be an accessible, affordable and open-source solution to better understand interactions in urban environments.
+
+To that end, ODC has created a governance structure through which users should utilize in order to abide by the ODC ethical code. This document serves to define this code of ethics, with the objectives of maintaining integrity and trustworthiness through:
+
+*	Protection of the public interest
+*	Demonstration of personal integrity
+*	Showing respect for collaborators
+
+The ODC community is an open, safe and inclusive space fostering sharing and collaboration regardless of age, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual orientation. 
+
+This Code of Conduct applies as a condition of use but we expect everyone in our community to follow it. 
```

Comment:
I'm not a lawyer but this sentence makes it sound like it's part of the End User License Agreement, which as far as I understand is not what has been discussed in https://github.com/opendatacam/opendatacam/issues/390#issuecomment-842417970

> After taking a few days to reflect and spoke with @b-g also, decided to just include in a prominent way something like a "artist statement" on the website / documentation of OpenDataCam for next release... no re licensing, no code of conduct.. keep it very simple as always..